iothub_client/inc/iothub_client_options.h

```diff
@@ -52,10 +52,23 @@ extern "C"
     static STATIC_VAR_UNUSED const char* OPTION_PRODUCT_INFO = "product_info";
 
     /*
-    * @brief    Specifies the Digital Twin Model Id of the connection. Only valid for use with MQTT Transport
+    * @brief Specifies the Digital Twin Model Id of the connection. Only valid for use with MQTT Transport
     */
     static STATIC_VAR_UNUSED const char* OPTION_MODEL_ID = "model_id";
 
+    /*
+    * @brief Determines the format to send in the username, which will inform IoT Hub what format
+    *        device expects the twin document to use. If this option si not set, the default is
```

Comment:
```suggestion
    *        device expects the twin document encoding use. If this option is not set, the default is
```

Reply:
I think I want to stay away from the word "encoding" here, since that can also refer to UTF-8, UTF-16, etc specifically, and system properties already differentiates between content type and content encoding..

Reply:
ok